plugins/inventory/proxmox.py

```diff
@@ -360,7 +362,17 @@ def _get_vm_config(self, properties, node, vmid, vmtype, name):
                         agent_iface_key = self.to_safe('%s%s' % (key, "_interfaces"))
                         properties[agent_iface_key] = agent_iface_value
 
-                if config not in plaintext_configs and not isinstance(value, int) and all("=" in v for v in value.split(",")):
+                if config == "lxc":
```

Comment:
Since mode of the code above uses single quotation marks, I think it's better to change this to:
```suggestion
                if config == 'lxc':
```

Reply:
Integrated.